src/UiElement/Registry.php

```diff
@@ -31,7 +31,7 @@ public function addUiElement(UiElementInterface $uiElement): void
         Assert::keyNotExists($this->uiElements, $uiElement->getCode(), 'UiElement with code "%s" is already registered.');
 
         $this->uiElements[$uiElement->getCode()] = $uiElement;
-        if (null !== $uiElement->getAlias()) {
+        if (null !== $uiElement->getAlias() && $uiElement->isEnabled()) {
```

Comment:
I think we want to keep the alias here. It's important to be able to parse the elements in a JSON.
We just want to avoid displaying them, we do not want any error in old contents. I mean, we could re-enable the UI Element later…